core/chains/evm/gas/block_history_estimator.go

```diff
@@ -855,7 +855,7 @@ func (b *BlockHistoryEstimator) EffectiveGasPrice(block evmtypes.Block, tx evmty
 	switch tx.Type {
 	case 0x0, 0x1:
 		return tx.GasPrice
-	case 0x2:
+	case 0x2, 0x3:
```

Comment:
I had a question on the approach.
Do we want to use the blob type txs for gas estimations, or just ignore them?
I think I remember reading that the fees for blob txs were going to be different than regular 0x2 ones, so this may pollute our data, if we use 0x3 for estimations